Source/AwsCommonRuntimeKit/crt/ByteBuffer.swift

```diff
@@ -72,7 +72,7 @@ extension ByteBuffer: IStreamable {
 }
 
 extension ByteBuffer {
-    func encodeToHexString() -> String {
-        data.map { String(format: "%02x", $0) }.joined()
+    public func encodeToHexString() -> String {
+        data.encodeToHexString()
     }
 }
```

Comment:
Trivial: I would prefer if we can remove this from ByteBuffer class to make it more simple. We can always do `byteBuffer.getData().encodeToHexString()`

Reply:
sgtm! i'll remove it then!